apps/preview/next/pages/examples/SfScrollable.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from 'classnames';
+import { SfScrollable, SfScrollableDirection, SfScrollableButtonsPlacement } from '@storefront-ui/react';
+import { prepareControls } from '../../components/utils/Controls';
+import ComponentExample from '../../components/utils/ComponentExample';
+import { ExamplePageLayout } from '../examples';
+
+function Example() {
+  const { state, controls } = prepareControls(
+    [
+      {
+        type: 'text',
+        modelName: 'children',
+        description: 'Only for demonstration purposes. Default slot, replaces example scrollable content',
+      },
+      {
+        type: 'select',
+        modelName: 'direction',
+        options: Object.keys(SfScrollableDirection),
+        propType: 'enum',
+        propDefaultValue: SfScrollableDirection.horizontal,
+        description: 'Direction of scrollable',
+      },
+      {
+        type: 'select',
+        modelName: 'buttonsPlacement',
+        options: Object.keys(SfScrollableButtonsPlacement),
+        propType: 'SfScrollableButtonsPlacement',
+        propDefaultValue: SfScrollableButtonsPlacement.blocked,
+        description: 'Change position of next/previous buttons',
+      },
+      {
+        type: 'text',
+        modelName: 'as',
+        propType: 'string',
+        propDefaultValue: 'div',
+        description: 'Tag of scrollable HTML element',
+      },
+      {
+        type: 'boolean',
+        modelName: 'drag',
+        propType: 'boolean',
+        propDefaultValue: false,
+        description: 'Enables item dragging on desktop',
+      },
+      {
+        type: 'boolean',
+        modelName: 'snap',
+        propType: 'boolean',
+        propDefaultValue: false,
+        description: 'Only for demonstration purposes. Enables item snapping, works only if `drag=true`',
```

Comment:
`Snap` seems to be working as single option as well. BTW, why is supposed to work only along with a `drag` option?

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2571/commits/c7c2922fa2e3340d0e114ff6f7e6bc0af74387a4